packages/core-ui/src/modules/insights/components/chart/TableRenderer.tsx

```
@@ -112,7 +113,7 @@ const TableList = (props: Props) => {
       <ChartTable>
         <thead>
           <tr>
-            {(headers || []).map(header => (
+            {(headerRow || []).map(header => (
               <>{renderSorter(header)}</>
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)